docs/action_network.rst

```diff
@@ -1,18 +1,36 @@
 Action Network
 ==========
 
+********
+Overview
+********
+
 `Action Network <https://actionnetwork.org/>`_ is an online tool for storing information
-about and organizing volunteers and donors. For more information, see `Action Network developer docs <https://actionnetwork.org/docs>`_
+about and organizing volunteers and donors. It is used primarily for digital organizing and event mangement. For more information, see `Action Network developer docs <https://actionnetwork.org/docs>`_
+
+.. note::
+  Authentication
+  Only ActionNetwork partners are able to access their API. You can generate your key from the API & Sync page, located in the "Start Organizing" menu, in the right column.
 
 ***********
 Quick Start
 ***********
 
+To instantiate a class, you can either pass in the API token as an argument or set the ``AN_API_TOKEN`` environmental variable.
+
 .. code-block:: python
-	
-	from parsons import ActionNetwork, Table
 
-	an = ActionNetwork(API_TOKEN)
+   from parsons import VAN
```

Comment:
Should import ActionNetwork, not VAN.

I'm guessing this was copied from ActionKit, where I previously introduced this exact typo. Maybe you can fix the ActionKit bug in your PR too, especially since it is referenced as an example for documentation?

Reply:
Good catch. And, yes, your theory is correct.

I've actually got this fixed in the S3 docs PR for AK, so I think that it should be fixed shortly there too.